contracts/upgradeable_contracts/arbitrary_message/MessageProcessor.sol

```diff
@@ -47,46 +47,50 @@ contract MessageProcessor is EternalStorage {
         addressStorage[MESSAGE_SENDER] = _sender;
     }
 
+    function messageId() public view returns (bytes32) {
+        return bytes32(uintStorage[MESSAGE_ID]);
+    }
+
     function transactionHash() external view returns (bytes32) {
```

Comment:
Should we add a comment to mention that this method is kept for backward compatibility? The same question applies to `failedMessageDataHash` and `setFailedMessageDataHash`

Reply:
Yes, it makes sense. I added a doc comments in 2b04c78